apps/deploy_monitoring/src/configuration.rs

```diff
@@ -0,0 +1,258 @@
+// Copyright (c) SimpleStaking and Tezedge Contributors
+// SPDX-License-Identifier: MIT
+
+use std::path::{Path, PathBuf};
+
+use clap::{App, Arg};
+
+pub struct DeployMonitoringEnvironment {
+    // logging level
+    pub log_level: slog::Level,
+
+    // slack bot token
```

Comment:
@adonagy 
I would put this slack stuff to separate struct:
`struct SlackConfiguration` and also do it as optional here:
```
...
slack: Option<SlackConfiguration>
...
```
I want to run it locally without sending messages to any slack